### PR TITLE
N UI 5070 add ut for wizard v2

### PR DESCRIPTION
**Pull request by DmytroPutrin**

### CODE REVIEW

File: packages/bits/src/lib/wizard-v2/wizard-step-header/wizard-step-header.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+            it("should call createStepStateConfigMap method", () => {
+                const spy = spyOn(component, "createStepStateConfigMap" as never);
+
+                component.ngOnChanges(config);
+                expect(spy).toHaveBeenCalled();
+            });
+        });
+
+        describe("stringLabel", () => {
+            afterEach(() => {
```
Comment (lukasholcik) [inline packages/bits/src/lib/wizard-v2/wizard-step-header/wizard-step-header.component.spec.ts:74]: Why are these necessary? Each unit test runs a separate instance of the component, so having this sort of cleanup shouldn't be required.

Comment (DmytroPutrin) [reply packages/bits/src/lib/wizard-v2/wizard-step-header/wizard-step-header.component.spec.ts]: you are right, thnaks

**Review by dmytrofirsov — APPROVED** [review]

**Review by Vita-Karas — APPROVED** [review]

**Review by dmytrognatyshyn — APPROVED** [review]
